_posts/a-larasati/2020-09-01-larasati-reflection.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author: a-larasati
+title: "Week 1 Reflection"
+---
+Admittedly, I was nervous about this whole "learning to code" process, but it's that excited kind of nervous. 
+
+Our first lecture helped a little in terms of orienting myself to the tools of the trade for the class, and I finally got to use my Github account again. I've had it for a while but have never really done anything with it, so it'll be fun to actually see that activity chart turn green ever so slightly.
+
+During the in-class exercises, I really had to keep myself from looking at other people and comparing what they've created on Trinket to mine. It's one of my fatal flaws, but then again, imposter syndrome is a very real thing that many people face. I had to remind myself to be patient and to not compare myself to what others can do. 
+
+As I was working through the take-home exercises, I realized I had to take a slightly different approach in how I should find information that will help me write the necessary code. Looking at Chapter 2, there was so much information for me to go through that I knew from the get-go that reading through everything would simply overwhelm me, so I switched to looking at what the exercises actually want me to do. Obviously the material covered in the chapter is important, but I thought that it would be a bit of an information overload if I read first then worked on the assignments. So I looked at the assignments first: the instructions, the specific pieces of information needed, what the code should actually do. This helped streamline the process tremendously for me because now I have a big picture of what needs to be done, and I can start sifting through the reading material to see what can point me in the right direction. Once the exercises were taken care of, reading through the material was less stressful.
+
+There were some moments of frustration while reading through the material and moments of "I need info about X but there's only Y and Z mentioned." Understanding the layout of the chapter was also an aha! moment for me (the '>>>' threw me off until I realized it just indicated the line of code that goes in the input). Trinket kept losing connection while I was reading the chapter as well, sometimes not showing up at all on the page, so there were ??? moments without an example but the almighty Google helped in a pinch. I also got to experience the whole "everyone does the same thing a little differently" firsthand so that's a plus.
+
+It has been quite some time since I've done the type of problem thinking/problem solving style that coding requires. It very much reminds me of the math classes I took during high school and even during my undergraduate degree. Can't say I love math, but I didn't hate it either. Also can't deny the fact that I did a little dance when I got the code to work the way I want it to. It's the little things, right?
```

Comment:
Indeed it is. Doing a little dance is a not uncommon and too seldom talked about programming activity.